plugins/woocommerce-blocks/tests/e2e/tests/cart/cart-checkout-block-notices.shopper.classic_theme.spec.ts

```diff
@@ -60,8 +61,7 @@ test.describe( 'Shopper → Classic Notice Templates', () => {
 			} )
 		).toBeVisible();
 
-		// We're explicitly checking for the following CSS classes here:
-		// .woocommerce-notices-wrapper .woocommerce-message
+		// We're explicitly checking the CSS classes.
```

Comment:
I'm not sure this comment (and the other instances like it) are adding much value, giving the reason why we are checking them would be more useful.